package/MDAnalysis/core/groups.py

```diff
@@ -665,7 +713,7 @@ class AtomGroup(object):
 
     """
 
-    level = 'atom'
+    order = 'group'
```

Comment:
And besides readability those compound names have to match the keys of the classdict returned by `make_classes()`, which in turn have to match the hardcoded names in `Universe._process_attr` and `TopologyAttr.transplants`.

Maybe in the future we can overhaul the naming scheme, possibly in a way that incorporates `order` into `level.Level()`, but it'll involve some refactoring...